### PR TITLE
new fallback for piamPackages

### PR DESCRIPTION
**Pull request by pfuehrlich-pik**

### CODE REVIEW

**Review by caviddhen — APPROVED** [review]

File: R/piamPackages.R [hunk 1]
```diff
@@ -7,12 +7,18 @@
...
+                "GDPuc", "gdx", "gdxrrw", "gms", "goxygen", "HARr", "iamc", "limes",
+                "lpjclass", "lpjmlkit", "lucode2", "luplot", "luscale", "lusweave",
+                "m4fsdp", "madrat", "magclass", "magpie4", "MagpieNCGains", "magpiesets",
+                "mip", "modelstats", "mrcommons", "mrdieter", "mrdrivers", "mredgebuildings",
+                "mrfable", "mrfeed", "mrfish", "mrland", "mrmagpie", "mrremind",
+                "mrsoil", "mrtutorial", "mrvalidation", "mrvalidnitrogen", "mrwaste",
+                "mrwater", "mrwaterplots", "mstools", "piamenv", "piamInterfaces",
+                "piamModelTests", "piktests", "quitte", "r2databus", "regressionworlddata",
+                "remind2", "remulator", "rmndt", "rOpenscmRunner", "shinyresults",
+                "SPARQL", "trafficlight")
```
Comment (mikapfl) [inline R/piamPackages.R:20]: Is there a way to generate this list during build time, e.g. during buildLibrary() or so?

Comment (pfuehrlich-pik) [reply R/piamPackages.R:20]: This is precisely what we had before, but for some reason for David this didn't work.

Comment (pfuehrlich-pik) [reply R/piamPackages.R:20]: okay that's not quite right

Comment (pfuehrlich-pik) [reply R/piamPackages.R:20]: We had during building lucode2 before (this PR changed that), but not during buildLibrary. Although before that I believe we even had it in buildLibrary

Comment (pfuehrlich-pik) [reply R/piamPackages.R:20]: Could probably be restored to that state, but as far as I can see this is not too important right now?

Comment (mikapfl) [reply R/piamPackages.R:20]: nah, if it was not working before, let's not overcomplicate this.

**Pull request merged**